source/ui/ui_public.h

```diff
@@ -185,6 +186,11 @@ typedef struct
 	void ( *L10n_LoadLangPOFile )( const char *filepath );
 	const char *( *L10n_TranslateString )( const char *string );
 	const char *( *L10n_GetUserLanguage )( void );
+
+	// steam
+	void *( *Steam_OpenProfile )( uint64_t steamid );
+
+	blockentry_t (*( *GetBlocklist )( void ))[MAX_BLOCKS];
```

Comment:
Here it is a function to access the array, in the other export it is a direct array?